backend/src/resolvers/users.spec.js

```diff
@@ -5,15 +5,15 @@ import Factory from '../seed/factories'
 const factory = Factory()
 let client
 
-afterAll(async () => {
+afterEach(async () => {
```

Comment:
@ulfgebhardt `afterEach` means: Run this after each test case. `afterAll` means: Run this after the entire test **suite**. `afterAll` is great for performance tweaks, but you have to tear down the created data yourself. Read: https://jestjs.io/docs/en/setup-teardown